src/api/db/data/20250916123855_add_value_count_to_anitya_distribution_attribute.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddValueCountToAnityaDistributionAttribute < ActiveRecord::Migration[7.2]
+  def up
+    ans = AttribNamespace.find_by_name('OBS')
+    at = AttribType.find_by(attrib_namespace: ans, name: 'AnityaDistribution')
+    at.update_column(:value_count, 1)
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
Could we find a way to make this data migration reversible? Is it really needed we mark it as irreversible?

I guess something like this should work (not tested):
```ruby
  def up
    ans = AttribNamespace.find_by_name('OBS')
    at = AttribType.find_by(attrib_namespace: ans, name: 'AnityaDistribution')
    at.update_column(:value_count, nil)
  end
```